Dragonflight/PaladinRetribution.lua

```diff
@@ -1255,7 +1255,7 @@ spec:RegisterAbilities( {
 
     -- Talent: Reduces all damage you take by $s1% for $d.
     divine_protection = {
-        id = 403876,
+        id = 498,
```

Comment:
No, Divine Protection is both 403876 and 498.  It swaps to 403876 when Aegis of Protection is talented.